recipes/octo-logger-cpp/all/conanfile.py

```diff
@@ -0,0 +1,45 @@
+from conans import ConanFile, CMake
+from conan.tools.files import get, collect_libs
+
+
+class OctoLoggerCPPConan(ConanFile):
+    name = "octo-logger-cpp"
+    version = "1.0.0"
+    license = "MIT"
+    url = "https://github.com/ofiriluz/octo-logger-cpp"
+    homepage = "https://github.com/ofiriluz/octo-logger-cpp"
+    description = "Octo logger library"
+    topics = ("logging", "cpp")
+    author = "Ofir Iluz"
```

Comment:
```suggestion
```
All recipes as maintained by all Conan community, so we don't add `author` attribute.

Reply:
gotcha, removed